LiveStream/ViewModels/LiveVideoViewModel.swift

```diff
@@ -42,6 +48,15 @@ internal protocol LiveVideoViewModelOutputs {
 
   /// Emits a stream when the subscriber of that stream should be removed.
   var removeSubscriber: Signal<OTStreamType, NoError> { get }
+
+  /// Emits when all subscribers should be re-subscribed when the view reappears.
+  var resubscribeAllSubscribersToSession: Signal<(), NoError> { get }
+
+  /// Emits to toggle play/pause when the view disappears/reappears.
+  var toggleHlsPause: Signal<Bool, NoError> { get }
```

Comment:
we've had this naming discussion before, but it was a loooong time ago. basically, `toggleABC` is maybe not the right description of this since that implies it takes the current state of the player and flips it to the opposite state.

we could do `setHlsPlaybackState: Signal<Bool, NoError>`?